_content/tour/rus/maps.article

```diff
@@ -1,24 +1,23 @@
-Maps
-A map is a data structure that provides support for storing and accessing data based on a key.
+Карты
+Карта - это структура данных, которая обеспечивает поддержку хранения и доступа к данным по ключу.
```

Comment:
```suggestion
Карта (отображение, мапа) - это структура данных, которая обеспечивает поддержку хранения и доступа к данным по ключу.
```

Let's mention all the variations once just in case.